src/main/java/com/github/tonivade/resp/protocol/RedisSerializer.java

```diff
@@ -142,6 +135,7 @@ private void growBuffer(int len) {
       byte[] array = new byte[buffer.position()];
       ((Buffer) buffer).rewind();
       buffer.get(array);
+      buffer = ByteBuffer.allocate(INITIAL_CAPACITY);
```

Comment:
Here renew buffer maybe is not best way. If so, the buffer will cause gc.
My option: 
```
        public void clear() {
            if (buffer.capacity() != INITIAL_CAPACITY) {
                buffer = ByteBuffer.allocate(INITIAL_CAPACITY);
            } else {
                buffer.clear();
            }
        }
```

And not invoke build in growBuffer, supply another way to copy data to grow buffer.

Reply:
:+1: